example/static_images.js

```diff
@@ -88,14 +88,23 @@ $(function() {
                 length: 10
             },
             decoder : {
-                readers : ["code_128_reader"]
+                readers : ["code_128_reader"],
+                drawScanline: true,
+            },
+            numOfWorkers: 4,
+            locator: {
+                halfSample: false,
```

Comment:
Tells the locator to use the full-sized image instead of a scaled down version. Should be set to ``false`` in this case. This is only necessary to be ``true`` if the barcode is really small and hard to be recognized.